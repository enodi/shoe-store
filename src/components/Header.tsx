import { Box, Heading } from "@chakra-ui/react";

const Header = () => (
  <Box sx={{ bg: "gray.400" }}>
    <Heading sx={{ textAlign: "center", p: 3, color: "#FFF" }}>
      Aldo Store
    </Heading>
  </Box>
);

export default Header;
