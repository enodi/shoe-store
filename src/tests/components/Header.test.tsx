import { shallow } from "enzyme";
import Header from "../../components/Header";

describe("Header Component", () => {
  let wrapper: any;
  beforeEach(() => {
      wrapper = shallow(<Header />);
  });

  it("should render Header component correctly", () => {
      expect(wrapper).toMatchSnapshot();
  });

  it("should contain Heading element with text", () => {
      expect(wrapper.find("Heading").text()).toEqual("Aldo Store");
  });
});