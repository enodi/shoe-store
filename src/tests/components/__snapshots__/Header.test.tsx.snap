// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render Header component correctly 1`] = `
<Box
  sx={
    Object {
      "bg": "gray.400",
    }
  }
>
  <Heading
    sx={
      Object {
        "color": "#FFF",
        "p": 3,
        "textAlign": "center",
      }
    }
  >
    Aldo Store
  </Heading>
</Box>
`;
